README.md

```diff
@@ -29,7 +29,25 @@ Supports utility projects that do not compile an assembly.
 
 Supports staging artifacts from build outputs.
 
-## What Are MSBuild SDKS?
+## How can I use these SDKs?
+
+Start your project file with:
```

Comment:
Its probably worth really calling out here the fact that you **MUST** specify a version in order for the MSBuild project SDKs to be resolved.  I think that's the main point that is missing.  So whatever phrasing you think makes that clear would be good to add to this.